soccer/src/soccer/robot.cpp

```diff
@@ -353,94 +326,6 @@ void OurRobot::face(rj_geometry::Point pt) {
 }
-#pragma mark Robot Avoidance
-
-void OurRobot::reset_avoid_robot_radii() {
-    for (size_t i = 0; i < kNumShells; ++i) {
-        intent().opp_avoid_mask[i] =
-            (i == context_->their_info.goalie) ? *opp_goalie_avoid_radius : *opp_avoid_radius;
-    }
-}
-
-void OurRobot::approach_all_opponents(bool enable) {
-    for (float& ar : intent().opp_avoid_mask) {
-        ar = (enable) ? kOppAvoidSmall : static_cast<float>(*opp_avoid_radius);
-    }
-}
-void OurRobot::avoid_all_opponents(bool enable) {
-    for (float& ar : intent().opp_avoid_mask) {
-        ar = (enable) ? -1.0f : static_cast<float>(*opp_avoid_radius);
-    }
-}
-
-bool OurRobot::avoid_opponent(unsigned shell_id) const {
-    return intent().opp_avoid_mask[shell_id] > 0.0;
-}
-
-bool OurRobot::approach_opponent(unsigned shell_id) const {
-    return avoid_opponent(shell_id) && intent().opp_avoid_mask[shell_id] < kRobotRadius - 0.01;
-}
-
-float OurRobot::avoid_opponent_radius(unsigned shell_id) const {
-    return intent().opp_avoid_mask[shell_id];
-}
-
-void OurRobot::avoid_opponent(unsigned shell_id, bool enable_avoid) {
-    if (enable_avoid) {
-        intent().opp_avoid_mask[shell_id] = *opp_avoid_radius;
-    } else {
-        intent().opp_avoid_mask[shell_id] = -1.0;
-    }
-}
-
-void OurRobot::approach_opponent(unsigned shell_id, bool enable_approach) {
-    if (enable_approach) {
-        intent().opp_avoid_mask[shell_id] = kOppAvoidSmall;
-    } else {
-        intent().opp_avoid_mask[shell_id] = *opp_avoid_radius;
-    }
-}
-
-void OurRobot::avoid_opponent_radius(unsigned shell_id, float radius) {
-    intent().opp_avoid_mask[shell_id] = radius;
-}
-
-void OurRobot::avoid_all_opponent_radius(float radius) {
-    for (float& ar : intent().opp_avoid_mask) {
-        ar = radius;
-    }
-}
-
-#pragma mark Ball Avoidance
-
-void OurRobot::disable_avoid_ball() { avoid_ball_radius(-1); }
-
-void OurRobot::avoid_ball_radius(float radius) {
-    intent().avoid_ball_radius = radius;
-
-    cmd_text_ << "avoid_ball(" << radius << ")" << endl;
-}
-
-float OurRobot::avoid_ball_radius() const { return intent().avoid_ball_radius; }
-
-void OurRobot::reset_avoid_ball() { avoid_ball_radius(kBallAvoidSmall); }
-
-std::shared_ptr<rj_geometry::Circle> OurRobot::create_ball_obstacle() const {
-    // if game is stopped, large obstacle regardless of flags
-    if (context_->game_state.state != GameState::Playing &&
-        !(context_->game_state.our_restart || context_->game_state.their_penalty())) {
-        return std::make_shared<rj_geometry::Circle>(
-            context_->world_state.ball.position,
-            FieldDimensions::current_dimensions.center_radius());
-    }
-
-    // create an obstacle if necessary
-    if (intent().avoid_ball_radius > 0.0) {
-        return std::make_shared<rj_geometry::Circle>(context_->world_state.ball.position,
-                                                    intent().avoid_ball_radius);
-    }
-    return nullptr;
-}
```

Comment:
Is this removed because it is unused?

Reply:
It's been unused for a while but this PR also removed the corresponding useless items from `RobotIntent`.